docs/integrations/unit-testing.md

```diff
@@ -9,38 +9,27 @@ Unit testing should be used to test small units of code in an isolated and deter
 
 In order to work with unit testing, the integration or automation script needs to be developed in [package (directory) structure](package-dir), where the yml file is separated from the python file and resides in its own directory.
 
-### Setup Pipenv
-
-To run locally the unit tests we want to setup a virtual environment with all required dependencies (both runtime and development). To achieve this we use [Pipenv](https://pipenv.readthedocs.io/en/latest/).
-
-Our recommended way to setup an integration into the [package (directory) structure](package-dir) is to use: `demisto-sdk split`. See full command documentation [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/split/README.md).
-
-Manual Setup:
-
-* **Install pipenv**: Follow the [instructions](https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv).
-* **Copy base Pipenv files**: Copy the base Pipfile and Pipfile.lock files to the target package directory from: [demisto_sdk/commands/lint/resources](https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/lint/resources).
-* **Install additional runtime dependencies**: using: `pipenv install <dependency>`. For example: `pipenv install ldap3`
-* **Sync Pipenv**: (including dev dependencies) by running: `pipenv sync --dev`
-* **Enable Virtual Env**: To enable the Pipenv virtual env in the shell run: `pipenv shell`. To exit the virtual env simply run: `exit`.
-
-You should now have a managed virtual environment to run unit tests locally.
+### General Notes
+In order to verify that the content indeed runs with all the required dependencies, we recommend writing, running, and debugging the unit tests locally with the corresponding image. 
+The VSCode Cortex XSOAR Extension allows you to do this in the easiest way possible. Additionally, you can run and debug the unit tests locally with the corresponding image in other IDEs.
 
 ### Setup Vscode
```

Comment:
```suggestion
### Setup Dev Environment 
```

Reply:
We need to distinguish between the setup of the the user can setup the dev environment with Vscode or with Pycharm